test/UnitTest/Microsoft.AspNetCore.OData.Test/PublicApi/Microsoft.AspNetCore.OData.PublicApi.bsl

```diff
@@ -2373,7 +2383,7 @@ public enum Microsoft.AspNet.OData.Query.AllowedQueryOptions : int {
 	Select = 4
 	Skip = 32
 	SkipToken = 256
-	Supported = 1279
+	Supported = 1535
```

Comment:
>Supported = 1535 [](start = 1, length = 16)

This means that, if an existing service that specifies "Supported" recompiles with the new library, without changing any other code, they will get the new behavior. We need to make sure we document this clearly.

Reply:
Not really, they will still have to enable the skiptoken. Previously skiptoken would have been ignored, now we will throw an error that it is not enabled and present in the query.

---
In reply to: [296940787](https://github.com/OData/WebApi/pull/1724#discussion_r296940787) [](ancestors = 296940787)